.github/workflows/build_linux_jax_wheels.yml

```
@@ -0,0 +1,117 @@
+name: Build Linux PyTorch Wheels
```

Update workflow name to match the file name
```suggestion
name: Build Linux JAX Wheels
```

Also, are these builds "portable", e.g. by building under a manylinux docker container? I recently renamed some workflows to show that more clearly: https://github.com/ROCm/TheRock/pull/1023. It doesn't look like it from a read of https://github.com/ROCm/rocm-jax/blob/master/build/ci_build, so are these Linux packages only runnable on Ubuntu (or whichever distro is used for the build)?

---

Yes, these are built under a manylinux wheel. For a number of reasons, our build scripts are a little oniony. But we create a manylinux image with ROCm here, and then do the build in that: https://github.com/ROCm/rocm-jax/blob/master/jax_rocm_plugin/build/rocm/ci_build#L57